councilmatic_core/models.py

```diff
@@ -531,7 +531,7 @@ def organization(self):
 
     @property
     def current_member(self):
-        if self.memberships:
```

Comment:
The membership object returns true even when there are no members, and so "self.memberships.first()` returns `None`, and the call for `membership.end_date` errors out